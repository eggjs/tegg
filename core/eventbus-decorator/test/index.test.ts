import { expect, test } from 'vitest';

import * as types from '../src/index.ts';

test('should export stable', async () => {
  expect(types).toMatchSnapshot();
});
