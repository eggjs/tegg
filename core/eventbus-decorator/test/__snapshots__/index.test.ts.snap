// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`should export stable 1`] = `
{
  "CORK_ID": Symbol(eventBus#corkId),
  "EVENT_CONTEXT_INJECT": Symbol(EggPrototype#event#handler#context#inject),
  "EVENT_NAME": Symbol(EggPrototype#eventName),
  "Event": [Function],
  "EventContext": [Function],
  "EventInfoUtil": [Function],
}
`;
